.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
   Rust and Foreign Language tests:
     docker:
       - image: rfkelly/uniffi-ci:latest
-    resource_class: medium
+    resource_class: large
```

Comment:
Looks like `rondpoint` has started reliably exceeding the resources available to a medium instance. I'm going to bump it to a large for now, but that's probably a sign that it needs to be split up.

Reply:
ouch

Reply:
yeah :-/